website/blog/2025-10-15-introducing-atmos-auth-list.md

```
@@ -0,0 +1,187 @@
+│     │  │  └─ arn: arn:aws:iam::123456789012:role/AdminRole
+│     │  └─ ops (aws/assume-role) [ALIAS: ops-admin]
+│     │     └─ Principal
+│     │        └─ arn: arn:aws:iam::987654321098:role/OpsRole
+│     └─ developer (aws/assume-role) [ALIAS: dev]
+│        └─ Principal
+│           └─ arn: arn:aws:iam::123456789012:role/DeveloperRole
+└─ okta (okta)
+   └─ URL: https://example.okta.com
+```
+
+**Automation Integration**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)